packages/terra-responsive-element/docs/README.md

```diff
@@ -7,7 +7,7 @@ The viewport can be set to the immediate parent or window.
 
 It is not necessary to set each breakpoint. If a breakpoint is not set the component will look at each successive smaller breakpoint until one is found.
 
-The ResponsiveElement observes the available width. Meaning padding, borders, and margins set on the parent are not included in the calculation. Modifying box-sizing will impact how these properties are used in an elements width calculation. For more details see: https://css-tricks.com/almanac/properties/b/box-sizing/
+The ResponsiveElement observes the total width. Meaning padding, borders except the margins set on the parent are included in the calculation. Modifying box-sizing will impact how these properties are used in an elements width calculation. For more details see: https://css-tricks.com/almanac/properties/b/box-sizing/
```

Comment:
Do modifying the box-sizing still impact how these properties are used in an elements width calculation? cc/ @StephenEsser

Reply:
Yes, we need this to be mentioned in the documentation. as the default value of box-sizing property for the elements will be ' 'content-box' this includes the content, padding and the border in width calculation. if we change box-sizing value to 'border-box' then only the content width will get retrieved without border and padding.